### ISSUE TITLE
Documentation for Ruby remote debugging is not up-to-date

### ISSUE THREAD

**Issue by blfgomes** [Komodo/KomodoEdit#1708]
### Short Summary
The newest documentation for Ruby remote debugging (http://docs.komodoide.com/manual/debugruby) is not accurate.

### Steps to Reproduce
The documentation states that the line below is sufficient for opening the program with the debugger stopped at the first line:
(after setting the _dbgdir_ and _RUBYDB_OPTS_ variables)

$ ruby -I"$dbgdir" -r"$dbgdir"/rdbgp2.rb <Program_To_Debug.rb>

But in reality, the steps described in the rdbgp2.rb file were what actually made remote debugging work for me:

 _This client is a stand-alone Ruby application that directly debugs client code
 via the Byebug debugger while communicating with an IDE. There are two ways
 the IDE (or user) can use this client:
   (1) Envoke this script with the name of the file to debug as its only
       argument. Any options like host and port must be specified as
       `key=value`, space-separated strings in a `RUBYDB_OPTS` environment
       variable. See the end of this script for more details.
   (2) `require` this script from the file to debug and call
           Byebug::DBGP.start()
       above the line to start debugging on. Keyword arguments for setting the
       host and port (especially for remote debugging) and other options are
       available. See that method's documentation for more details._

### Platform Information

*Komodo IDE 10.0* 
*Red Hat Enterprise Linux Server release 7.2 (Maipo)* 
*ruby 2.0.0p598 (2014-11-13) [x86_64-linux]*

### Additional Information

Unless I missed something, the documentation at http://docs.komodoide.com/manual/debugruby no longer applies. Besides that, method 2) - call to Byebug::DBGP.start() - is not well documented, since the method **requires** an argument. The hint _See that method's documentation for more details_ provided some guidance, but it is not ideal.

**Comment by mitchell-as** [comment]
Thanks for reporting this and for all the detective work you did! It is much appreciated.

**Comment by mitchell-as** [comment]
Updated the Ruby Debug page (http://docs.komodoide.com/Manual/debugruby) to provide the correct remote debugging shell command for using the new `rdbgp2.rb` library with Ruby 2.1+. (I also made it more clear which commands should be run for Ruby 1.8-2.0 vs. Ruby 2.1+.)

I chose not to address the `Byebug::DBGP.start()` way of initiating debugging (method 2) because it's not something we want to officially support. It's for internal documentation purposes only.
